.github/workflows/build.yml

```diff
@@ -148,16 +151,19 @@ jobs:
     - name: Activate VS cmd (arm)
       uses: ilammy/msvc-dev-cmd@v1
       with:
-        arch: x86_arm
+        arch: amd64_arm
```

Comment:
why

Reply:
Why not? All other builds use amd64...

Reply:
I'm not sure, but since we're using host-tools built for x86 architecture in case of cross-compilation, there might be a slight boost (of build speed) when running on the same architecture.

Reply:
> slight boost (of build speed)

I don't know, not sure it would matter much.
Also, `x86_arm` means msvc itself uses x86 not amd64: why would we want that for arm?

Reply:
Moreover, https://reactos.org/wiki/ARM_Port does read `amd64_arm`...

Reply:
I tested this originally on `x86_arm`, which is why that is what is being used.